tests/test_define.h

```diff
@@ -17,10 +17,14 @@ extern "C" {
 #define TRYTES_81_2                                                            \
   "RVORZ9SIIP9RCYMREUIXXVPQIPHVCNPQ9HZWYKFWYWZRE9JQKG9REPKIASHUUECPSQO9JT9XNM" \
   "VKWYGVA"
+#define TRYTES_81_3                                                            \
+  "FPPBU9AOXZHSJEWDCPWGCPKGOVIUWWPDTIIXDJUKYYNNPTWCGDWFBZLKVCLOKGYEHJUMABSTA9" \
+  "OP99999"
 #define BUNDLE_HASH                                                            \
   "LVXEVZABVCIFEDSCONKEVEYBSIRMXGHLJDKSKQHTKZC9ULEAPSLKOOWCCZJGWSIISDDSEVUQHV" \
   "GPFOSIW"
 #define TAG_MSG "TANGLEACCELERATOR9999999999"
+#define FIND_TAG_MSG "TANGLEACCELERATORSEND999999"
```

Comment:
Why `SEND`?

Reply:
We can modify to the other one.
Because I use `TANGLEACCELERATORSEND999999` when sending :see_no_evil: 
Maybe change it to `TANGLEACCELERATORFORFINDTXN`?

Reply:
It is too long to me. Maybe `TAFINDTXN` would be cleaner.